lib/themes/dosomething/paraneue_dosomething/js/utilities/ApiClient.js

```diff
@@ -36,6 +43,37 @@ class ApiClient {
   }
 
   /**
+   * Send a DELETE request to the given path URI.
+   *
+   * @param  {String} path
+   * @return {Object}
+   */
+  delete(path) {
+    const url = `${this.url}${path}`;
+
+    // @TODO: see todo in post() method!
+    console.log('%c ApiClient DELETE Request: \n' + url,
```

Comment:
Nope. On purpose!

> Random, but also trying out leaving in a couple useful console.log() calls (and coloring them) with the ApiClient so that during local development, we can see what activity is taking place when making API calls. These logs get pulled out during builds on Production!